### PR TITLE
add example for duty-cycling

### PR DESCRIPTION
**Pull request by greyghost509**
Example based on calcout records that run two different timers to process actions/records in a duty-cycle manner.

### CODE REVIEW

**Comment by gabrielfedel** [comment]
Nice example! could you just rename the db for example3.db ? I notice that the 2nd example is not following the "standard" name for database, so I will fix that too

**Comment by greyghost509** [comment]
Am Mon, 27 Nov 2023 01:13:18 -0800
schrieb Gabriel Fedel ***@***.***>:

> Nice example! could you just rename the db for example3.db ? I notice
> that the 2nd example is not following the "standard" name for database,
> so I will fix that too

I just pushed a rename for the example file. Looks like I got confused
about the numbering (starting at 0 vs 1 :-).


cu
  Gerrit

File: 3/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# 
+# The length of the timers are set via DUTY_TIM1 and DUTY_TIM2.
+# Your can set different values (in this case 37 for TIM2) using
+
+$EPICS_BIN/caput DUTY_TIM2 37
+```
+
+## Files
+
+* [example4.db](example4.db)
```
Comment (gabrielfedel) [inline 3/README.md]: And here also please :)

**Comment by greyghost509** [comment]
Am Mon, 27 Nov 2023 02:03:33 -0800
schrieb Gabriel Fedel ***@***.***>:

> +* [example4.db](example4.db)

> And here also please :)


Oops, I missed that one, sorry. Fixed now.


cu
  Gerrit

**Comment by gabrielfedel** [comment]
Great! It is merged! Thanks for the contribution!

**Comment by greyghost509** [comment]
Am Mon, 27 Nov 2023 02:20:09 -0800
schrieb Gabriel Fedel ***@***.***>:

> Great! It is merged! Thanks for the contribution!

You're welcome. I'll send in more examples if I run across something that
looks generic enough so other people might benefit from it.

**Comment by gabrielfedel** [comment]
Please do! I will send an email to tech talk to talk about this repository soon.

**Pull request merged**